src/main/java/ui/components/KeyboardShortcuts.java

```diff
@@ -0,0 +1,116 @@
+package ui.components;
+
+import javafx.scene.input.KeyCode;
+import javafx.scene.input.KeyCodeCombination;
+import javafx.scene.input.KeyCombination;
+import prefs.Preferences;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * a central place to specify keyboard shortcuts
+ *
+ * Classes that currently have keyboard shortcut code:
+ * ui.components.NavigableListView
+ * ui.issuecolumn.ColumnControl
+ * ui.issuepanel.IssuePanel
+ * ui.MenuControl
+ *
+ * Utility Class:
+ * util.KeyPress
+ */
+public class KeyboardShortcuts {
```

Comment:
For example, if i map the key <kbd>L</kbd> to both navigate to the left panel and open a new issue, and the new issue mapping occurs later in the json, so it will take on the user defined shortcut key. my question is, what happens to the key mapping of navigating to the left panel? do we use the default because this will have its own problems too, like if the default key is mapped to some other action.